docs/en_US/Tutorial/SearchSpaceSpec.md

```diff
@@ -45,15 +45,16 @@ All types of sampling strategies and their parameter are listed here:
   * When optimizing, this variable is constrained to a two-sided interval.
 
 * {"_type":"quniform","_value":[low, high, q]}
-  * Which means the variable value is a value like round(uniform(low, high) / q) * q
+  * Which means the variable value is a value like randint(floor((high-low)/q)+1)*q + low. For example, for _value specified as [0, 10, 2.5], possible values are [0, 2.5, 5.0, 7.5, 10.0]; For _value specified as [2, 10, 5], possible values are [2, 7]. All possible values have the same possibility to be selected.   
```

Comment:
Also need to update here https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/AnnotationSpec.md

Reply:
updated